### DISCUSSION TITLE
How to show text in edit form?

### DISCUSSION THREAD

**Discussion by divdax** [filamentphp/filament#5744] [Help] [answered]
How to show a column in the edit form as plain text instead as a field with readonly?

**Comment by josefbehr** [comment, marked as answer]
Either use ->disabled() or use a Placeholder field to only show something inside a field layout without any possibility to edit.

**Reply by divdax** [reply]
I need a placeholder. Thank you, @josefbehr! The documentation on how to set a model column for the content is not well-documented, but I was able to figure it out.

```php
Placeholder::make('Filesize')
    ->content(fn ($record) => $record->filesize),
```
